src/core/core.scale.js

```diff
@@ -1017,18 +1010,10 @@ var Scale = Element.extend({
 				continue;
 			}
 
-			if (i === me.zeroLineIndex && options.offset === offsetGridLines) {
-				// Draw the first index specially
-				lineWidth = gridLines.zeroLineWidth;
-				lineColor = gridLines.zeroLineColor;
-				borderDash = gridLines.zeroLineBorderDash || [];
-				borderDashOffset = gridLines.zeroLineBorderDashOffset || 0.0;
-			} else {
-				lineWidth = valueAtIndexOrDefault(gridLines.lineWidth, i, 1);
-				lineColor = valueAtIndexOrDefault(gridLines.color, i, 'rgba(0,0,0,0.1)');
-				borderDash = gridLines.borderDash || [];
-				borderDashOffset = gridLines.borderDashOffset || 0.0;
-			}
+			lineWidth = valueAtIndexOrDefault(gridLines.lineWidth, i, 1);
```

Comment:
you could make this be `const lineWidth =`

Reply:
Done